API_CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ file.
 
 This file will be updated weekly, if there are notable changes.
 
+## 2018-06-25
+
+## Checkout
+
+* Added `transfered` Payment State
+* Correctly add Line Items for deposits referenced by other line items
+* Added `createdAt` date to [Checkout Info](api_checkout.md#checkout-info)
+
+## Products
+
+* REMOVED Deprecated `decimalDigits` and `currency` in
+  [Product](api_products.md#product-object)
+
+* FIXED: Fixed several bugs in the creation of the appdb, which could
```

Comment:
schöner: `FIXED several bugs in appdb creation, which could lead to missing products`